bia_export/cli.py

```diff
@@ -8,20 +7,23 @@
 
 from bia_integrator_api import models as api_models
 
-
-logging.basicConfig(
-    level="NOTSET", format="%(message)s", datefmt="[%X]", handlers=[RichHandler()]
+from .data_mapping_utils import (
+    transform_ai_study_dict,
+    transform_so_study_dict,
+    transform_study_dict,
+    create_export_image,
+    get_ann_uuid_by_sourcename,
 )
-logger = logging.getLogger()
 
-from .scli import (
+from .bia_client_utils import (
     rw_client,
```

Comment:
I assumed the point of `bia_client_utils` was to prevent a direct dependence of cli on `rw_client` ...